src/index.js

```diff
@@ -20,6 +20,7 @@ var ImageMapView = Backbone.View.extend({
         var self = this;
         this.template = options.template;
         this.items = new models.ItemList(options.items);
+        this.background = options.background;
```

Comment:
Does `options.background` need a default value, i.e. do the callers of this library need to update their `options` object?

Reply:
Good note. Callers of the library do not need to update their options if they choose to handle background rendering in a different way.